### DISCUSSION TITLE
Non-Docker Installation

### DISCUSSION THREAD

**Discussion by taylorn-ai** [Donkie/Spoolman#145] [Q&A]
How would I go about setting this up on an existing web server, without docker? I’d like to run it on a Raspberry Pi that I’m already using for something else.

**Comment by Disane87** [comment]
Have a look into the `DOCKERFILE`, there is a section which runs the application at the end (be aware of the dependencies):
https://github.com/Donkie/Spoolman/blob/master/Dockerfile
